package.json

```diff
@@ -45,67 +50,77 @@
     "release:major": "node scripts/release/release.js major",
     "release:minor": "node scripts/release/release.js minor",
     "release:patch": "node scripts/release/release.js patch",
-    "test": "npm run jest && npm run compile && npm run nightwatch && npm run nightwatch:port-provided && wdio wdio.conf.js",
-    "wdio": "wdio wdio.conf.js"
+    "test": "npm run jest && npm run nightwatch && npm run nightwatch:port-provided && npm run wdio",
+    "wdio": "npm run wdio-webpack-obj && npm run wdio-webpack-func && npm run tt-wdio-webpack-obj && npm run tt-wdio-webpack-func",
+    "wdio-webpack-obj": "wdio",
+    "wdio-webpack-func": "TT_TEST_WDIO_FUNCTION=true wdio",
```

Comment:
Curious, what do the following scripts do:`wdio-webpack-func` and `tt-wdio-webpack-func`?

Reply:
We can take both a webpack obj and webpack function as configuration that we pass to the webpack compiler. When passing an object, we have to clone it before setting the webpack ouptut dir https://github.com/cerner/terra-toolkit/pull/113/files#diff-2d0e39e2b5431dc8d9df655f5fc376fbR49

Reply:
These scripts are testing webpack configuration passed as a function which is toggle in wdio.config [here](https://github.com/cerner/terra-toolkit/pull/113/files#diff-c2b12f3e29d36b46344876dbf97bd0f8R6)